src/Api/Auth/Controllers/AccountsController.cs

```diff
@@ -255,6 +255,23 @@ public async Task PostVerifyEmail()
         await _userService.SendEmailVerificationAsync(user);
     }
 
+    [HttpPost("request-sm-access")]
+    public async Task RequestSMAccessFromAdmins([FromBody] RequestSMAccessRequestModel model)
+    {
+        if (model?.EmailContent == null || model?.OrganizationId == null)
+        {
+            throw new BadRequestException();
+        }
```

Comment:
What about adding validation attributes on the request model instead? 
https://learn.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-8.0#validation-attributes

Reply:
Sounds good to me 😄